docs/source/usage.rst

```diff
@@ -71,3 +71,35 @@ The format is JSON, with a single array containing each todo as a single entry
 given field, it will be printed as ``null``.
 
 Fields MAY be added in future, but will never be removed.
+
+Sorting
+-------
+
+The tasks can be sorted with the --sort argument. Sorting may be done according to the following fields:
```

Comment:
Please use <code>\`\`</code> around "literal text" such as option names or field names. Otherwise they may be wrongly rendered.